projects/CMakeLists.txt

```diff
@@ -22,8 +22,8 @@
 
 # Include Project Directories
 add_subdirectory(common)
-add_subdirectory(tools)
+# add_subdirectory(tools)
 add_subdirectory(authserver)
```

Comment:
As per the description, we are only building the authserver and common for this draft pull request, just so that we can make quick progress and figure out a plan for the whole project.